.github/workflows/master.yml

```diff
@@ -33,7 +33,21 @@ jobs:
   r:
     runs-on: ubuntu-latest
     steps:
+    - name: Uninstall default-jdk and adoptopenjdk-11-hotspot if present
+      run: |
+       sudo apt-get -y remove --purge default-jdk adoptopenjdk-11-hotspot || :
```

Comment:
@yitao-li Thanks for the PR. Can you insert a comment on why we need this?

Reply:
Sure!